test/testUtils/FormTester.scala

```diff
@@ -14,55 +13,102 @@ class FormTester(options: Map[String, (String, String)], smallCheckbox: String="
 
     optionsToSelectToGenerateFormErrors
   }
-
-  def checkHtmlForOptionAndItsAttributes(htmlAsString: String,
-                                         optionsSelected: Map[String, String]=Map(),
-                                         formStatus: String="NotSubmitted"): Unit = {
+  //scalastyle:off cyclomatic.complexity
+  def checkHtmlForOptionAndItsAttributes(htmlAsString: String, optionsSelected: Map[String, String], formStatus: String="NotSubmitted"): Unit = {
```

Comment:
Yes, it was like [this initially](https://github.com/nationalarchives/tdr-transfer-frontend/pull/2063/commits/32b7e5a3d87b158d3549a1f75f151c11f0b923ae) but I changed it here in a commit in this PR because the static/dynamic forms work slightly differently: empty `Map()` for static forms means no value was selected but empty `Map()` for dynamic forms means that something went wrong, since the Map should always contain the fields whether you select anything or not; so I thought it might be less confusing if it wasn't an empty Map by default. What do you think?

Reply:
Yeah, that makes sense. I would create an overloaded method where the map is not required. Since it's tests so it's fine, we can pass an empty `Map()` wherever we need to.